docs/user/build-customization.rst

```diff
@@ -1,30 +1,30 @@
-Build customization
-===================
+Build process customization
+===========================
 
-Read the Docs has a :doc:`well-defined build process <builds>` that works for many projects,
-but we offer additional customization to support more uses of our platform.
-This page explains how to extend the build process using :term:`user-defined build jobs` to execute custom commands,
-and also how to override the build process completely:
+Read the Docs has a :doc:`well-defined build process </builds>` that works for many projects.
+We also allow customization of builds in primary ways:
 
 `Extend the build process`_
-    If you are using Sphinx or Mkdocs and need to execute additional commands.
+    This is useful if you want to use our deafult build process,
+    but just run additional commands.
 
 `Override the build process`_
-    If you want full control over your build. This option supports any tool that generates HTML as part of the build.
-
-.. contents:: Table of contents
-   :local:
+    This option gives you *full control* over your build.
+    Read the Docs supports any tool that generates HTML.
 
 Extend the build process
 ------------------------
 
 In the normal build process,
 the pre-defined jobs ``checkout``, ``system_dependencies``, ``create_environment``, ``install``, ``build`` and ``upload`` are executed.
-However, Read the Docs exposes extra jobs to users so they can customize the build process by running shell commands.
-These extra jobs are:
+Read the Docs also exposes additional ujobs,
+which allow you to customize the build process by running shell commands.
```

Comment:
```suggestion
Read the Docs also exposes these jobs,
which allows you to customize the build process by adding shell commands.
```

Reply:
Bleh, not sure why my editor didn't flag these typos 🙃